src/ansys/dpf/core/helpers/streamlines.py

```diff
@@ -40,11 +45,110 @@ def __init__(self, data):
+        nodes_scoping.ids = np.arange(1, data_set.n_points + 1)
+        streamlines_field = dpf.Field(location=locations.nodal)
+        fdef = streamlines_field.field_definition
+        fdef.name = array_names[0]
+        streamlines_field.scoping = nodes_scoping
+        streamlines_field.data = data_arrays[0]
+        mesh = dpf.MeshedRegion()
+        coords_field = dpf.Field(location=locations.nodal)
+        coords_field.scoping = nodes_scoping
+        coords_field.data = points_array
+        mesh.set_coordinates_field(coords_field)
+        elems_scoping = dpf.Scoping(location=locations.elemental)
+        elems_scoping.ids = np.arange(1, data_set.n_cells + 1)
+        connectivity_field = dpf.PropertyField(location=locations.elemental)
+        # connectivity size is different for each element,
+        # data array can't be set in once
+        for ind, dat in enumerate(cell_points):
+            connectivity_field.append(dat, ind + 1)
+        elems_types_field = dpf.PropertyField(location=locations.elemental)
+        elems_types_field.scoping = elems_scoping
+        elems_types_field.data = cell_types_converted
+        mesh.nodes.coordinates_field = coords_field
+        mesh.elements.connectivities_field = connectivity_field
+        mesh.elements.element_types_field = elems_types_field
+        streamlines_field.meshed_region = mesh
+        fc_to_return = dpf.FieldsContainer()
+        fc_to_return.add_label(DefinitionLabels.time)
+        fc_to_return.add_field({DefinitionLabels.time: 1}, streamlines_field)
+
+        return fc_to_return
 
     def _fc_to_pv_data_set(self):
         """Convert FieldsContainer into pyvista.PolyData."""
-        raise Exception("Not implemented yet")
+        fields = self._streamlines_fc
+
+        from ansys.dpf.core.vtk_helper import PyVistaImportError
+        try:
+            import pyvista as pv
+        except ModuleNotFoundError:
+            raise PyVistaImportError
+        import vtk
+
+        streamlines_field = fields.get_field({DefinitionLabels.time: 1})
+        mesh = streamlines_field.meshed_region
+        cell_types = mesh.elements.element_types_field.data
+        cell_points = []
+        conn_field = mesh.elements.connectivities_field
+        for i in range(len(cell_types)):
+            cell_points.append(conn_field.get_entity_data(i))
+        points = mesh.nodes.coordinates_field.data
+        array_names = [ streamlines_field.field_definition.name ]
+        data_arrays = [ streamlines_field.data ]
+
+        ncells = len(cell_types)
+        npoints = len(points)
+        vpoly = vtk.vtkPolyData()
+        vpoints = vtk.vtkPoints()
+        vpoints.SetNumberOfPoints(npoints)
+        vtk_array_points = pv.convert_array(arr=points)
+        vpoints.SetData(vtk_array_points)
+        vpoly.SetPoints(vpoints)
+        vcells = vtk.vtkCellArray()
+        for i in range(0, ncells):
+            this_cell_points = cell_points[i]
```

Comment:
Maybe this could be improved using a method that sets the whole connectivity data array and the whole cell type array, however I did not find it (yet ?).